Maple2.Model/Metadata/Server_Table/InstancedFieldTable.cs

```diff
@@ -0,0 +1,6 @@
+using System.Collections.Generic;
+using Maple2.Model.Enum;
+
+namespace Maple2.Model.Metadata;
+
+public record InstanceFieldTable(IReadOnlyDictionary<int, InstanceType> Entries) : ServerTable;
```

Comment:
I think we should be parsing more than just the instance type, no? 
the instanceID may be useful. Not sure but it's definitely in the packet.